README.md

```diff
@@ -1,3 +1,5 @@
+<p align="center"><img src="https://raw.githubusercontent.com/jbeguna04/findbugs-android/master/LogoDesign/logotype1.png"></p>
```

Comment:
Please use the markdown syntax for image with relative urls
Eg.

    ![Logo](LogoDesign/logotype1.png)